### PR TITLE
Intern Task Alina Mulik: Homework

### PR DESCRIPTION
**Pull request by alina-mulik**
Added Hometask 6 files

### CODE REVIEW

File: MainHospitalClass/Hospital.cs [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 namespace MainHospitalClass
 {
-    public class Hospital
+    public class Hospital : Institution
     {
         private const int PerVisitPayment = 500;
         private string _name;
```
Comment (yauhenists) [inline MainHospitalClass/Hospital.cs]: You could create protected properties in Base class (for _budget as well) and use it here

File: People/Person.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            Id = id;
+        }
+
+        public string GetName()
+        {
+            if (gender.ToLower() == "female")
+            {
+                return "Mrs. " + name;
+            }
+            else if (gender.ToLower() == "male")
```
Comment (yauhenists) [inline People/Person.cs]: Not required "else" here and below as you have "return"

File: People/Patient.cs [hunk 3]
```diff
@@ -0,0 +1,99 @@
+﻿namespace People
+{
+    public class Patient : Person
+    {
+        private bool isSick;
```
Comment (yauhenists) [inline People/Patient.cs:5]: Missing underscore for private fields Names

File: People/Doctor.cs [hunk 4]
```diff
@@ -48,26 +40,27 @@ public int GetSalary()
...
-        public void AddPatientsToDoctorProfile(string[] patients)
+        public void AddPatientsToDoctorProfile(Patient[] patients)
         {
             if (_patientsArray != null)
             {
-                foreach (string patient in patients)
+                foreach (Person patient in patients)
                 {
                     if (_patientsArray.Contains(patient))
                     {
```
Comment (yauhenists) [inline People/Doctor.cs]: Not required "continue" statement

**Review by yauhenists — CHANGES_REQUESTED** [review]

File: MainHospitalClass/Hospital.cs (hunk 1 above)
Comment (alina-mulik) [reply MainHospitalClass/Hospital.cs]: Good point 😅 Fixed.

File: People/Person.cs (hunk 2 above)
Comment (alina-mulik) [reply People/Person.cs]: Fixed.

File: People/Patient.cs (hunk 3 above)
Comment (alina-mulik) [reply People/Patient.cs:5]: Oops, fixed it.

File: People/Doctor.cs (hunk 4 above)
Comment (alina-mulik) [reply People/Doctor.cs]: Removed. Thanks!

**Review by yauhenists — APPROVED** [review]

**Pull request merged**